LRAC-6088 Update yarn.lock

fcodias opened the pull request.

fcodias committed:
LRAC-6088 Use Map in contexts storage

fcodias committed:
LRAC-6088 Create new hash fn

fcodias committed:
LRAC-6088 Remove spread polyfill

fcodias committed:
LRAC-6088 Use unfetch as fetch polyfill

fcodias committed:
LRAC-6088 Remove Math polyfill

fcodias committed:
LRAC-6088 Update entry polyfills

fcodias committed:
LRAC-6088 Standardize UUID

fcodias committed:
LRAC-6088 Hash email on identity request

fcodias committed:
LRAC-6088 Update tests

fcodias committed:
LRAC-6088 Remove webpack plugin
ModuleConcatenationPlugin was deprecated on webpack.optimize and now is default on production mode

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Masterfoni commented on modules/apps/analytics/analytics-client-js/src/eventQueue.js:
```diff
@@ -281,23 +286,27 @@ class EventQueue {
-					return;
-				}
+			if (!events) {
+				return;
```
Hey @fcodias, would you agree on moving the `promisesArr.push` bit of code inside this `if` clause? And ofc, check for `events` instead of `!events`? I think this change could make it a bit easier to read

Masterfoni commented on modules/apps/analytics/analytics-client-js/src/utils/map.js:
```diff
@@ -0,0 +1,35 @@
+ */
+
+function convertMapToArr(mapInstance) {
+	const arr = [];
```
In the cases that we need to use this function, could we use `Array.from` instead or we're limited because of IE?

fcodias replied:
It's a IE11 limitation :/

Masterfoni commented on modules/apps/analytics/analytics-client-js/src/utils/map.js:
```diff
@@ -0,0 +1,35 @@
+	const arr = [];
+
+	mapInstance.forEach((value, key) => {
+		arr.push(key);
```
Could we remove the curly brackets here and maybe have just one line?

kienD replied:
Couldn't we just use `map` instead and return the result?

fcodias replied:
We can't because `.map` isn't available fot `Map` objects

Masterfoni reviewed (approved):
Awesome @fcodias! 🤓 Just some questions, though none of the are critical. overall LGTM

fcodias committed:
LRAC-6098 Improve sort fn

fcodias committed:
LRAC-6088 SF

Masterfoni reviewed (approved):
LGTM

andreldm commented:
Code-wise it looks good to me, I would just try to abstract away the context retrieval and bookkeeping in a separated module (e.g. `contexts.js`) instead of dealing with `getContexts` and `setContexts` directly. This can be done later anyway.

kienD commented on modules/apps/analytics/analytics-client-js/src/utils/hash.js:
```diff
@@ -12,13 +12,34 @@
 	});
 
-export {hash};
+export {hash, legacyHash};
```
Can we use one style of function declaration?

andreldm commented:
@fcodias I noticed something weird using the client with these changes, after login, when navigating between pages, different `userId` are generated for each context (SPA enabled, both FF and Chrome).
This doesn't happen with the current version in production, can you please double check?

kienD commented on modules/apps/analytics/analytics-client-js/src/client.js:
```diff
@@ -206,13 +209,7 @@ class Client {
-										payload = item;
-									}
-
+								messages.map((payload) => {
```
We can probably remove the curly braces and just return it directly.

fcodias committed:
LRAC-6088 Fix generating multiples userIds

fcodias committed:
LRAC-6088 Extract contexts to new module

fcodias committed:
LRAC-6088 SF

fcodias commented:
Updated @andreldm @kienD

kienD reviewed (approved):
LGTM

fcodias committed:
LRAC-6088 Send email only on setIdentity

boton committed:
LPS-124975 Add first test render as expected without video

boton committed:
LPS-124975 Test render as expected with video

andreldm commented:
With the exception of the error when handling contexts generated by the current client version, LGTM.

alee8888 committed:
LPS-124813 Implement Country search

jkappler committed:
LPS-125568 Adds schema + endpoint to retrieve all the experiences for a given page

jkappler committed:
LPS-125568 Implement endpoints to get all experiences for a given page

jorgediaz-lr committed:
LPS-125637 Update css portlet id reference in portlet preferences

jorgediaz-lr committed:
LPS-125637 Update css portlet id reference for overridden cases

jkappler committed:
LPS-125568 Extract to DTOConverter since we are going to be using it in other modules

jkappler committed:
LPS-125568 Adds endpoint to get the definition of a specific experience of a content page

markocikos committed:
LPS-121853 Create new management toolbar tag based on Clay v3

jkappler committed:
LPS-125568 Adds endpoint instead of friendlyURL

jorgediaz-lr committed:
LPS-125637 Fix typo we are updating the old value into the database. This code was added in LPS-122562

jorgediaz-lr committed:
LPS-125637 Add missing paranthesis in the SQL because all the portletId conditions must be evaluated together. This code was added in LPS-122562

jorgediaz-lr committed:
LPS-125637 Combine the two StringUtil.replace calls into one for the base class

alee8888 committed:
LPS-124813 Add tests

alee8888 committed:
LPS-124813 build-service-portal

jorgediaz-lr committed:
LPS-125637 Combine the two StringUtil.replace calls into one for the overriden classes

4lejandrito committed:
LPS-125478 Allowing empty data definitions for document types

jorgediaz-lr committed:
LPS-125637 Minor fixes of LPS-122562 code: Added a 'distint' to the SQL to avoid processing each portletPreferencesId more than once. Get the portletPreferences object returned from updatePortletPreferences call to avoid using an outdate object

boton committed:
LPS-124975 Add default values to URL and HTML avoiding creating an uncontrolled input and later changing it to a controlled input

jorgediaz-lr committed:
LPS-125637 Avoid a concurrence problem during the PortletPreferences update in case they are in the same Layout because every change in a PortletPreference triggers an Layout update

4lejandrito committed:
LPS-125478 Adding missing DDM structures

Empty document types coming from 7.3 have no dataDefinitionId associated
(0).

In 7.4 there is a unique index for the dataDefinitionId field
therefore we can't have more than 1 document type with the same value
(0) for a given group.

With this upgrade we create a structure for each of those document types
so that there are no document types left without one (apart from BASIC
DOCUMENT).

We also recreate the index in case it failed because the DB was already
in an incompatible state (more than 1 document type with the same
dataDefinitionId (0)).

boton committed:
LPS-124975 Fetch the fields and render the HTMl iframe

boton committed:
LPS-124975 Avoid repetition and add test check the request url with param

boton committed:
LPS-124975 Check if the callback function is called

joseabelenda committed:
LPS-125684 - Update smallValue on PortletPreferenceValue

BryanEngler committed:
LPS-123770 portal-search-web: Display a message in the Search Bar configuration when the header Search Bar configurations are used

https://issues.liferay.com/browse/LPS-123770

BryanEngler committed:
LPS-123770 portal-search-web: buildlang

https://issues.liferay.com/browse/LPS-123770

fcodias committed:
LRAC-6088 Isolate upgradeStorage

fcodias commented:
ci:test:sf

alee8888 committed:
LPS-124813 Clean up Country Localization table after a Country is deleted

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 57 seconds</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/dab7ad790ee947a2abc6cc6b656a1caa46943ac5">dab7ad790ee947a2abc6cc6b656a1caa46943ac5</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/fcodias/liferay-portal/tree/LRAC-6088">LRAC-6088</a><br/>Branch GIT ID: <a href="https://github.com/fcodias/liferay-portal/commit/3e20128212edba4383cd059c2a083c2efff750a6">3e20128212edba4383cd059c2a083c2efff750a6</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-source-format/5204/"><strike><strong>test-portal-source-format</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-source-format/builds/5204/jenkins-report.html">here</a>.</h5><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/fcodias/liferay-portal/tree/LRAC-6088">fcodias/LRAC-6088</a></strong><pre><code>Target exception: java.lang.RuntimeException: Unable to rebase fcodias-40-LRAC-6088 to master
error: Failed to merge in the changes.</code></pre></p></div></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-21.liferay.com/job/test-portal-source-format/5204/">test-portal-source-format#5204</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-source-format/builds/5204/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/andreldm/liferay-portal/pull/40">andreldm#40</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2002.aspx">/Liferay DXP 7.4/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/79321.aspx">master - fcodias > andreldm - PR#40 - 2021-01-13[17:24:40]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-4.liferay.com/job/publish-spira-report/13146/">publish-spira-report#13146</a><br />

fcodias commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 0 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/dab7ad790ee947a2abc6cc6b656a1caa46943ac5">dab7ad790ee947a2abc6cc6b656a1caa46943ac5</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/fcodias/liferay-portal/tree/LRAC-6088">LRAC-6088</a><br/>Branch GIT ID: <a href="https://github.com/fcodias/liferay-portal/commit/d7ecec7c4ae2dd8759dfded826690ceacf697ef0">d7ecec7c4ae2dd8759dfded826690ceacf697ef0</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-19.liferay.com/job/test-portal-source-format/4991/"><strike><strong>test-portal-source-format</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-19.liferay.com/userContent/jobs/test-portal-source-format/builds/4991/jenkins-report.html">here</a>.</h5><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/fcodias/liferay-portal/tree/LRAC-6088">fcodias/LRAC-6088</a></strong><pre><code>Target exception: java.lang.RuntimeException: Unable to rebase fcodias-40-LRAC-6088 to master
error: Failed to merge in the changes.</code></pre></p></div></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-19.liferay.com/job/test-portal-source-format/4991/">test-portal-source-format#4991</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-19.liferay.com/userContent/jobs/test-portal-source-format/builds/4991/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/andreldm/liferay-portal/pull/40">andreldm#40</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2002.aspx">/Liferay DXP 7.4/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/79322.aspx">master - fcodias > andreldm - PR#40 - 2021-01-13[17:30:09]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-8.liferay.com/job/publish-spira-report/11805/">publish-spira-report#11805</a><br />

jkappler committed:
LPS-125568 We cannot render based on viewPorts so remove the property

jkappler committed:
LPS-125568 Update DTOConverter

fcodias commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 1 minute</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/dab7ad790ee947a2abc6cc6b656a1caa46943ac5">dab7ad790ee947a2abc6cc6b656a1caa46943ac5</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/fcodias/liferay-portal/tree/LRAC-6088">LRAC-6088</a><br/>Branch GIT ID: <a href="https://github.com/fcodias/liferay-portal/commit/d7ecec7c4ae2dd8759dfded826690ceacf697ef0">d7ecec7c4ae2dd8759dfded826690ceacf697ef0</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-source-format/5404/"><strike><strong>test-portal-source-format</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-23.liferay.com/userContent/jobs/test-portal-source-format/builds/5404/jenkins-report.html">here</a>.</h5><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/fcodias/liferay-portal/tree/LRAC-6088">fcodias/LRAC-6088</a></strong><pre><code>Target exception: java.lang.RuntimeException: Unable to rebase fcodias-40-LRAC-6088 to master
error: Failed to merge in the changes.</code></pre></p></div></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-23.liferay.com/job/test-portal-source-format/5404/">test-portal-source-format#5404</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-23.liferay.com/userContent/jobs/test-portal-source-format/builds/5404/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/andreldm/liferay-portal/pull/40">andreldm#40</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2002.aspx">/Liferay DXP 7.4/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/79548.aspx">master - fcodias > andreldm - PR#40 - 2021-01-14[06:26:16]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-20.liferay.com/job/publish-spira-report/9751/">publish-spira-report#9751</a><br />

ealonso committed:
LPS-125805 Removes no necessary methods

ealonso committed:
LPS-125805 Uses lambda

jkappler committed:
LPS-125568 Adds endpoint to get the rendered content for a specific experience of a content page

jkappler committed:
LPS-125568 buildR

boton committed:
LPS-124975 Check if the server does not support the video URL

ealonso committed:
LPS-125805 It should be version 2.0, so we are going to check the new form builder and not the old one

ealonso committed:
LPS-125805 Update de json structure to match with the new data engine

ealonso committed:
LPS-125805 Fix tests: Uses the new field types

ealonso committed:
LPS-125805 build.gradle

boton committed:
LPS-124975 Check the add button in the item selector component

boton committed:
LPS-124975 Check if adding a valid URL fire a event in the opener

boton committed:
LPS-124975 Fire submit instead of click to avoid jsdom error: Not implemented: HTMLFormElement.prototype.submit

More info:
https://github.com/jsdom/jsdom/issues/1937
https://github.com/testing-library/react-testing-library/issues/755

kevenleone committed:
LPS-125887 remove NoPermissionEntry from ViewEntriesApp and update App Props

kevenleone committed:
LPS-125887 Update PermissionContext

kevenleone committed:
LPS-125887 Update unit tests

kevenleone committed:
LPS-125887 Use PermissionTunnel to handle Loading and PermissionState before render main content

kevenleone committed:
LPS-125887 Hide the DDMForm until the PermissionRequest are completed

jkappler committed:
LPS-125568 Implement endpoint for getting the definition of a content page based on it's experience

ealonso committed:
LPS-125805 Uses new constants where it is possible

jkappler committed:
LPS-125568 Implement endpoint for getting the rendered content of a content page based on it's experience

ealonso committed:
LPS-125805 Change the field type to match with the new data engine editor

jkappler committed:
LPS-125568 Implement tests

ealonso committed:
LPS-125805 Now the decimal field is a numeric field with datatype double. After the changes to new data engine we have only one numeric field.

ealonso committed:
LPS-125805 Removes no longer available field types

jkappler committed:
LPS-125568 Adds action

boton committed:
LPS-124975 Not submit the form if is loading or without fields

boton committed:
LPS-124975 Check invalid with invalid url / response

AliciaGarciaGarcia committed:
LPS-125442 when the dataDefinitionId is used, the link on DDMt StructureLink between the ddm and the fileEntry is not needed and this entry is preventing the deletion of the fileEntryType when it is not used anymore

joseabelenda committed:
LPS-125684 - Code review

AliciaGarciaGarcia committed:
LPS-125442 search FileEntryType by dataDefinitionId

kevenleone committed:
LPS-125887 add onPermissionChange hook to handle PermissionState changes

AliciaGarciaGarcia committed:
LPS-125442 change method to add file entry type with dataDefinitionId

AliciaGarciaGarcia committed:
LPS-125442 deprecate method cause now the structureLink is not used to find the fileEntryTypes

brookedalton committed:
LRQA-63702 Move widgets to second column so search and results are visible

brookedalton committed:
LRQA-63702 Add reindex so asset category appears

BryanEngler committed:
LPS-123770 portal-search-web: Update test

https://issues.liferay.com/browse/LPS-123770

michaelhashimoto committed:
LRCI-1916 Remove unused BatchTestClassGroup#getTestCasePropertiesContent()

michaelhashimoto committed:
LRCI-1916 Rename getRelevantTestBatchRunPropertyQuery() to getTestBatchRunPropertyQuery()

michaelhashimoto committed:
LRCI-1916 Add test base dir to functional axis & segment test class groups

michaelhashimoto committed:
LRCI-1916 Always store poshi properties in test class objects because they will be gone when recalled

michaelhashimoto committed:
LRCI-1916 Add method to getTestBaseDirs() for functional batch test class groups

michaelhashimoto committed:
LRCI-1916 Make getDefaultTestBatchRunPropertyQuery into a protected method to be overridden

michaelhashimoto committed:
LRCI-1916 Remove unneeded class AppReleaseFunctionalBatchTestClassGroup

michaelhashimoto committed:
LRCI-1916 Add plugins functional batch test class group

michaelhashimoto committed:
LRCI-1916 Add test base dir when getting poshi test class group

michaelhashimoto committed:
LRCI-1916 Set all the queries for each test base dir within functional groups

michaelhashimoto committed:
LRCI-1916 Update how functional axis test class groups are created

michaelhashimoto committed:
LRCI-1916 Clean up leftover unused artifacts

Commit:
LPS-125793 Add DuplicateSegmentsExperienceMVCActionCommand

Commit:
LPS-125793 Extract common code to util class

Commit:
LPS-125793 Add tests

Sharryxue committed:
LRQA-63993 Update the macro to contain partial of LPS-125224

Sharryxue committed:
LRQA-63993 Add macro to set address for account

Sharryxue committed:
LRQA-63998 Add macro to remove account address

Hanlf committed:
LPS-123529 Update translation macros

Hanlf committed:
LPS-123529 Update test to view translation entry

Sharryxue committed:
LRQA-63993 Add macro to view account address

Sharryxue committed:
LRQA-63993 Add test to set default account address

Hanlf committed:
LPS-123529 Add translation locator and macro

Hanlf committed:
LPS-123529 Add tests about editing and deleting translation enties

Sharryxue committed:
LRQA-63998 Add test to delete account address

Commit:
LPS-125807 Update liferay-ckeditor to v4.14.1-liferay.16

**Release notes:**

[Full changelog](https://github.com/liferay/liferay-ckeditor/compare/v4.14.1-liferay.15...v4.14.1-liferay.16)

-   fix: LPS-125441 CKEditor font size selector flickers on mouse over ([\#148](https://github.com/liferay/liferay-ckeditor/pull/148))

Additionally this fixes [LPS-125441](https://issues.liferay.com/browse/LPS-125441)

wincent committed:
LPS-125980 Format .eslintignore

Just noticed the mixed indents (tabs and spaces) in .eslintignore, which
we can fix with by applying the same auto-formatting we already use for
.prettierignore.

Note that I changed the glob for the latter from `*.prettierignore` to
`.prettierignore`, because these files are dotfiles and not extensions.

jeyvison committed:
LPS-125887 Add permission verification to the layout rendering

jeyvison committed:
LPS-125887 No need to show and hide the component anymore

marcosapmf committed:
LPS-125895 make sure hidden input value will be a string

4lejandrito committed:
LPS-125971 Adding a separate method to update the custom meta tags

The problem is that we are using the same method from 2 different views
(Opengraph and Custom Meta Tags) and also from the copyLayout method
that is used when a page is published.

Both the Opengraph and the copyLayout methods don't send the DDM form
and therefore it ends up being deleted.

By seggregating the 2 behaviours in separate methods we will only update
the DDM form when coming from the Custom Meta Tags view.

marcosapmf committed:
LPS-125894 check if localizedValue is an array

4lejandrito committed:
LPS-125971 Autogenerated

4lejandrito committed:
LPS-125971 Calling the new method from the Custom Meta Tags view

4lejandrito committed:
LPS-125971 Deduplicating

4lejandrito committed:
LPS-125971 Using the new method inside the test

4lejandrito committed:
LPS-125478 Splitting in separate version numbers

4lejandrito committed:
LPS-125478 Creating JSON programmatically

joseabelenda committed:
LPS-125684 - SF

ling-alan-huang committed:
LPS-105380 Catch more cases

wincent committed:
LPS-125980 SF

I obviously didn't touch this file in the parent commit, but as it
includes a change to the source formatter, CI wants to check all the
things, and I need to fix this in order to get a green run.

gabrielwas committed:
LPS-124592 Check required flag between ddmStructure and ddmStructureLayout

ling-alan-huang committed:
LPS-105380 Add testcases

marcoscv-work committed:
LPS-125670 - Remove unneeded classes

michaelhashimoto committed:
LRCI-1916 Change how PropsUtil#clean() works to allow it to reset to base class properties

michaelhashimoto committed:
LRCI-1916 Update com.liferay.poshi.core to 1.0.13

markocikos committed:
LPS-121853 Apply new management toolbar based on Clay v3

brianchandotcom committed:
LPS-125478 Found more of these

brianchandotcom committed:
LPS-125478 match modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/upgrade/v3_4_1/UpgradeJournalArticleLocalization.java

brianchandotcom committed:
LPS-125568 SF

brianchandotcom committed:
LPS-125568 Sort

brianchandotcom committed:
LPS-125568 as used

pei-jung committed:
LPS-124813 Simply tests and add inactive country to the data set to make sure search results only include specified active countries

willnewbury committed:
ISOPS-151 Add missing translation to Journal Properties Transformer Listener

4lejandrito committed:
LPS-125478 Ignoring failure when creating the index

The index will exist if there were no duplicate entries before the
upgrade but it will not exist otherwise.

With this we will ignore failures when the index already exists in some
dbs.

4lejandrito committed:
LPS-125478 SF

willnewbury committed:
ISOPS-151 Add ISOPS to ci.properties

austinchiang committed:
LPS-123529 Wordsmith

austinchiang committed:
LPS-123529 Sort

liuxingye committed:
LRQA-64386 Update macro to assert video title when edting

liuxingye committed:
LRQA-64350 Assert iframe before selecting

liuxingye committed:
LRQA-64352 Make the name of the repository special

Jeremy-Ch committed:
LRQA-64355 Update test property

Sharryxue committed:
LRQA-63998 Add test to remove account address

Jeremy-Ch committed:
LRQA-64357 Remove the redundant steps

liuxingye committed:
LRQA-64362 Update tests to use the suitable characters

liuxingye committed:
LRQA-64362 Update test names

Jeremy-Ch committed:
LRQA-64364 Add property to skip instance

JoyceWang08211 committed:
LRQA-64356 Update macro to add document with image field

JoyceWang08211 committed:
LRQA-64356 Update case to use image field in document type

JoyceWang08211 committed:
LRQA-64360 Update case to select the uploaded image document

Commit:
LRQA-64390 Modify locator to make it robust

JoyceWang08211 committed:
LRQA-64360 Modify macro to assert view image in Image field

JoyceWang08211 committed:
LRQA-64360 Unquarantine case

4lejandrito committed:
LPS-124975 Reusing

4lejandrito committed:
LPS-124975 Fixing test names

4lejandrito committed:
LPS-124975 Pulling out precondition

ealonso committed:
LPS-125974 Removes no needed styles. It was caused by LPS-119456

Those styles are causing problem with cards, since we are checking on icon.jsp the rowCssClass, if it is not null then we are adding the correct styles to the cards, if not we are not adding the correct styles and then we are facing problems with styles, the cards are overlapped.

Code in icon.jsp

if (Validator.isNull(rowCssClass)) {
	rowCssClass = "card-page-item card-page-item-asset";
}

brianchandotcom committed:
LPS-125980 prep next

brianchandotcom committed:
LPS-125980 prep next

brianchandotcom committed:
LRCI-1916 prep next

brianchandotcom committed:
LRCI-1916 prep next

4lejandrito committed:
LPS-125478 Simplifying with failOnError = false

victorg1991 committed:
LPS-125965 Use a "-" as the subtitle to avoid having cards of different size (as it is done in the site selector)

victorg1991 committed:
LPS-125965 Don't show the sticker in the card

brianchandotcom committed:
LPS-125793 If this kind of constructor protection is a good idea, let's talk to Hugo and have him enforce it everywhere, but let's not add it unless we choose to make this universal - Hugo SF

brianchandotcom committed:
LPS-124813 Just inline, or rename the variable 'list' to 'countries'

brianchandotcom committed:
LPS-124813 SF... this whole block should be generated and removed

brianchandotcom committed:
LPS-124813 Minor SF

4lejandrito committed:
LPS-123890 Using original breadcrumb on all popup windows

boton committed:
LPS-125430 Only update description when it is empty

brianchandotcom committed:
LPS-125637 I removed the new lines because I do not believe that order matters. If I'm wrong, please send a revert. Thx.

fcodias committed:
LRAC-6088 Use emailAddressHashed instead of email

fcodias committed:
LRAC-6088 Flatten identity request object

fcodias committed:
LRAC-6088 Update yarn.lock

fcodias commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6e0d8f334f923d77ba3bc9959b589a189a00ade0">6e0d8f334f923d77ba3bc9959b589a189a00ade0</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/fcodias/liferay-portal/tree/LRAC-6088">LRAC-6088</a><br/>Branch GIT ID: <a href="https://github.com/fcodias/liferay-portal/commit/11fe8dafc6e54db25b881a4d379a45c2b47e39a6">11fe8dafc6e54db25b881a4d379a45c2b47e39a6</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-source-format/5117/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/5117/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-3.liferay.com/job/test-portal-source-format/5117/">test-portal-source-format#5117</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/5117/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/andreldm/liferay-portal/pull/40">andreldm#40</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2002.aspx">/Liferay DXP 7.4/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/81076.aspx">master - fcodias > andreldm - PR#40 - 2021-01-19[11:37:33]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-12.liferay.com/job/publish-spira-report/11310/">publish-spira-report#11310</a><br />

andreldm commented:
LGTM, please forward to bchan if there is nothing left.

fcodias commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
`ci:test:sf`

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 22 out of 22 jobs passed in 1 hour 35 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b30d0d32a13478154ff8a311cf10a3c1303e6f3d">b30d0d32a13478154ff8a311cf10a3c1303e6f3d</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/2b89e6be0ae0f6e12dec390930d3e9525006e687">2b89e6be0ae0f6e12dec390930d3e9525006e687</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459801/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376363/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/484636/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462334/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462331/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462336/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462332/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462330/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/484633/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 22 out of 22 jobs PASSED</strong></summary><details><summary><strong>22 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/7636/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/484635/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376362/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459801/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401873/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376363/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/484636/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/484637/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/422984/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/484632/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462333/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462334/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/422983/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/484634/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462331/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/459802/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462336/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462332/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462335/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462330/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/462329/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/484633/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7636/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pullrequest to `brianchandotcom`.

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/97901

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/7636/">test-portal-acceptance-pullrequest(master)#7636</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7636/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/andreldm/liferay-portal/pull/40">andreldm#40</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2004.aspx">/Liferay DXP 7.4/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/81457.aspx">master - fcodias > andreldm - PR#40 - 2021-01-20[04:59:59]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-4.liferay.com/job/publish-spira-report/13672/">publish-spira-report#13672</a><br />
